src/javascript/app/common/redirect_banner.js

```diff
@@ -37,7 +37,7 @@ const RedirectBanner = (() => {
         BinarySocket.wait('authorize', 'website_status', 'landing_company').then(() => {
             const eu_country = isEuCountrySelected(Client.get('residence')) || isEuCountrySelected(State.getResponse('website_status.clients_country'));
 
-            const client_account = Client.get('landing_company_shortcode') === 'maltainvest';
+            const client_account = Client.get('landing_company_shortcode') === 'maltainvest' || Client.get('landing_company_shortcode') === 'iom' || Client.get('landing_company_shortcode') === 'malta';
```

Comment:
Can we assign `Client.get('landing_company_shortcode')` into a reusable variable

Reply:
okay sure